packages/syntax-core/src/LinkButton/LinkButton.module.css

```diff
@@ -1,4 +1,5 @@
-.linkButton {
```

Comment:
@somethiiing we also need `.linkButton` here since it's the default styling https://syntax-git-linkbutton-add-rel-prop-for-best-practices-cambly.vercel.app/?path=/docs/components-linkbutton--docs

![image](https://github.com/Cambly/syntax/assets/127199/89bfafca-77f5-47bf-9ed8-0a2915ea4974)